package.json

```diff
@@ -37,19 +37,20 @@
 	"dependencies": {
 		"chalk": "^2.1.0",
 		"fancy-log": "^1.3.2",
-		"plugin-error": "^0.1.2",
+		"gulp": "^4.0.0",
```

Comment:
I don't understand why `gulp` would be added as a dependency here? Is it referenced in the codebase somewhere I'm not seeing?

Reply:
You might be right i think was drunk when i made the PR